docs/content/02_basics/babel-plugin.md

```diff
@@ -2,3 +2,119 @@
 title: "Babel Plugin"
 order: 7
 ---
+
+Prodo's babel plugin is used to create components and actions using a more concise syntax.
+
+# Installation
+
+To install the babel plugin, install the package `@prodo/babel-plugin` as a dev-depenedency:
+
+```bash
+npm install --save-dev @prodo/babel-plugin
+# or
+yarn install --dev @prodo/babel-plugin
+```
+
+# Configuration
+
+Before you use the babel plugin, you will need to ensure that you are using Babel to transpile your code. If you are using webpack, this can be done by passing your code to the [`babel-loader`](https://www.npmjs.com/package/babel-loader). If you are using Parcel, your code will be transpiled by Babel automatically, so you do not need to do anything.
+
+Once you have ensured that you are using parcel, you will need to configure Babel to include the babel plugin. This is done by adding `"@prodo"` to the list of plugins used by Babel, usually defined in a file called _.babelrc_.
+
+```json
+{
+  "plugins": ["@prodo"]
```

Comment:
I add no idea this worked :o we should maybe use this in our examples and CPA

Reply:
It might also be useful to show a full example of a `.babelrc` file for users who may be unfamiliar with the whole babel ecosystem.

Reply:
Yeah it's cool - they infer `preset` and `plugin` in the right place as well:

https://babeljs.io/docs/en/options#name-normalization

Sounds good - reckon it's enough to link them to an example, or would you want to include it in the docs page directly?

Reply:
I would think just include in page directly. Easier to just copy and paste the config.

Reply:
ping :D @tdawes